cove_oc4ids/templates/cove_oc4ids/additional_checks_table.html

```diff
@@ -10,26 +10,71 @@
       </tr>
     </thead>
     <tbody>
-      {% for type, values in additional_checks.items %}
-        {% if type == 'empty_field' %}
+      {% for check in additional_checks %}
+        {% if check.check_id == 'missing-values' %}
           <tr>
             <td>{% trans 'The data includes fields that are empty or contain only whitespaces. Fields that are not being used, or that have no value, should be excluded in their entirety (key and value) from the data' %}</td>
```

Comment:
Minor grammar point - we should be consistent with full stops at the end of error messages.